http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -83,11 +85,18 @@ private TokenValidator(Builder builder) {
      * @throws OidcException if the ID token is invalid
      */
     public VerifiedTokens parseAndVerifyToken(final String idToken, final String accessToken) throws OidcException {
+        return parseAndVerifyToken(idToken, accessToken, null);
+    }
+
+    public VerifiedTokens parseAndVerifyToken(final String idToken, final String accessToken, final String stateCookieValue) throws OidcException {
```

Comment:
I don't think the method signature needs to be changed. We can get the session cookie value using the `clientConfiguration` variable.

Reply:
OidcClientConfiguration does not retain a reference to OidcHttpFacade or
any object that references OidcHttpFacade, which is required to get access
the the request's cookies. The value must be passed in to the method.

Reply:
done.   OidClientConfiguration is not needed.